app/datatables/team_users_datatable.rb

```diff
@@ -16,9 +16,9 @@ def sortable_columns
     @sortable_columns ||= [
       'User.full_name',
       'User.email',
-      'UserTeam.created_at',
+      'UserTeam.role',
+      'User.confirmed_at',
```

Comment:
Why same values here?

Reply:
Because last column is "status" and it is actually the value of `confirmed_at.present?`